Terminal.Gui/Core/View.cs

```diff
@@ -2025,10 +2020,10 @@ public void FocusLast ()
 				return;
 			}
 
-			for (int i = tabIndexes.Count; i > 0;) {
+			for (var i = tabIndexes.Count; i > 0;) {
 				i--;
 
-				View v = tabIndexes [i];
+				var v = tabIndexes [i];
```

Comment:
I think this reduces readability.  There are compiler rules for `var` use that divide into 3 classifications:

- [csharp_style_var_for_built_in_types](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0007-ide0008#csharp_style_var_for_built_in_types) 
- [csharp_style_var_when_type_is_apparent](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0007-ide0008#csharp_style_var_when_type_is_apparent)
- [csharp_style_var_elsewhere](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0007-ide0008#csharp_style_var_elsewhere)

I think the first 2 are great but the third covers this instance.  It is when you can't tell by looking at the line what Type of variable you have allocated (without mousing over).  Old line you can tell that `tabIndexes` is views but new line is less clear.

 `csharp_style_var_elsewhere`

Reply:
`Roslynator` normally advice to use `Type` instead of `var` because is more faster and avoid confusing. I also use `var` a lot but change from `Type` to `var` wouldn't be necessary. But on overall I like the changes but I need review it careful.

Reply:
I've switched that back to an explicit type and adjusted the Rider rules to account for that rule (though it still insists it's "apparent").